src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -222,6 +222,9 @@ public function getConfigTreeBuilder()
 
         $this->addFormatSection($rootNode, 'formats', [
             'jsonld' => ['mime_types' => ['application/ld+json']],
+        ]);
+        $this->addFormatSection($rootNode, 'documentation_formats', [
+            'jsonld' => ['mime_types' => ['application/ld+json']], // Hydra documentation, admin and PWA generator
             'json' => ['mime_types' => ['application/json']], // Swagger support
             'html' => ['mime_types' => ['text/html']], // Swagger UI support
```

Comment:
HTML must be defined also in `addFormatSection` section to have the "documentation for any URL" feature working.